gtk-sys/Cargo.toml

```diff
@@ -33,6 +33,7 @@ shell-words = "0.1.0"
 tempdir = "0.3"
 
 [features]
+docs-rs = []
```

Comment:
@GuillaumeGomez can you remove this feature?

Reply:
Sure.